docker/Dockerfile.browse

```diff
@@ -0,0 +1,19 @@
+FROM node:4.4.4
+LABEL node-version="4.4.4"
```

Comment:
Do we really need the label? The node base image should provide a `NODE_VERSION` env if I'm not mistaken.